components/ingest-service/pipeline/publisher/event_publisher.go

```diff
@@ -0,0 +1,118 @@
+package publisher
```

Comment:
This sends all chef-server actions as events to the event-feed-service through the event-service.

Reply:
This is the main change in this PR. The rest is piping or deleting.